SwiftUI_DZ/SwiftUI_DZ/VerificationView/VerificationView.swift

```diff
@@ -0,0 +1,239 @@
+    
+    var body: some View {
+        NavigationView {
+            VStack {
+                gradientNavigationView
+                VStack(spacing: Constants.defaultSpacing) {
+                    letterImageView
+                    verificationTextView
+                    smsNumberView
+                    checkSmsTextVStack
+                    continueButtonView
+                    progressView
+                    Text(Constants.noSmsText)
+                    sendSmsButtonView
+                }
+                Spacer()
+            }
+        }
+        .navigationBarTitleDisplayMode(.large)
+        .toolbar {
+            ToolbarItem(placement: .principal) {
+                titleTextView
+            }
+        }
+        .navigationBarItems(
+            leading:
+                backButtonView
+        )
+        .onTapGesture {
+            dismissKeyboard()
+        }
+    }
+    
+    // MARK: - Private Properties
+    
+    @Environment(\.presentationMode) private var presentation
+    @StateObject private var verificationViewModel = VerificationViewModel()
+    @FocusState private var firstNumberIsFocused: Bool
+    @FocusState private var secondNumberIsFocused: Bool
+    @FocusState private var thirdNumberIsFocused: Bool
+    @FocusState private var fourNumberIsFocused: Bool
+
+    private var gradientNavigationView: some View {
+            Rectangle()
+                .fill(Color.clear)
+                .frame(height: 10)
+                .background(LinearGradient(
+                    colors: [.yellow.opacity(1),
+                             .red.opacity(0.7)],
+                    startPoint: .topLeading,
+                    endPoint: .bottomTrailing)
+                )
+                .padding(.bottom, 5)
+        }
+    
+    private var titleTextView: some View {
+        Text(Constants.titleText)
+            .font(.title)
+            .bold()
+            .foregroundColor(.white)
+    }
+    
+    private var backButtonView: some View {
+        Button(
+            action: {
+                self.presentation.wrappedValue.dismiss()
+            },
+            label: {
+                Image(systemName: Constants.backwardChevronImageName)
+                    .resizable()
+                    .frame(
+                        width: CGFloat(Constants.chevronWidth),
+                        height: CGFloat(Constants.chevronHeight)
+                    )
+                    .foregroundColor(.white)
+            }
+        )
+    }
+    
+    private var letterImageView: some View {
+        Image(Constants.letterImageName)
+            .resizable()
+            .frame(width: Constants.sizeImage, height: Constants.sizeImage)
+            .scaledToFill()
+    }
+    
+
+    private var verificationTextView: some View {
+        Text(Constants.verificationCodeText)
+            .font(Font.system(size: Constants.largeFontSize))
+    }
+
+    private var smsNumberView: some View {
```

Comment:
```suggestion
    private var smsNumberView: some View {
```

где то ты такие элементы называет по логике что это и прибавляешь вью, а где то тоже самое стеками, я бы наверное пришла к одному стилю